src/Tags/ResponsiveTag.php

```diff
@@ -92,4 +92,26 @@ private function getAttributeString(): string
                 return $name . '="' . $value . '"';
             })->implode(' ');
     }
+
+    private function includePlaceholder(): bool
+    {
+        $includePlaceholder = $this->params->get('placeholder');
+
+        if (isset($includePlaceholder)) {
+            return $includePlaceholder;
+        }
+
+        return config('statamic.responsive-images.placeholder');
+    }
+
+    private function includeWebp(): bool
+    {
+        $includeWebp = $this->params->get('webp');
+
+        if (isset($includeWebp)) {
+            return $includeWebp;
+        }
+
+        return config('statamic.responsive-images.webp');
```

Comment:
Might want to add a fallback, just in case a config file gets messed up or they add `null`

```suggestion
        return config('statamic.responsive-images.webp', true);
```

Reply:
Good point.